src/performance-autotuning/index.md

```diff
@@ -32,3 +32,11 @@ bucket_size_2p,is_hierarchical_reduce,score,train_iter
 ```
 
 `bucket_size_2p` is the power of 2 of the bucket size, for example `bucket_size_2p=23` means bucket_size is 8388608 bytes ($2^{23}$).
+
+## Case Study
```

Comment:
Follow the section heading format. Only the first character of the first world should be capitalized.